CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ add_subdirectory(tests)
 include_directories(${MONSTER_WAVES_DIR})
 
 # Create executable file
-add_executable(MonsterWaves main.cpp)
+add_executable(MonsterWaves main.cpp "MonsterWaves/Entity/Player.cpp")
```

Comment:
@JakubMastalski co to tu robi? Może to powodować problemy
- [ ] wycofać zmiany z tego pliku